packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -60,7 +66,7 @@ Do not use actionable data lists when actions are restricted to a single row or
 ### Expandable data list
 The expandable list adds an expansion panel to every row to reveal more details about the item.
 
-![expandable data list](./img/expandable-list.png)
+<img src="./img/expandable-list.png"  alt="Expandable data list"  width="1174"/> 
```

Comment:
Would get rid of the "bulk selects" in the expanded table - cause I'm not sure how those would interact with the main bulk select haha